src/pages/library/dataBrowser-utils.js

```diff
@@ -106,6 +108,17 @@ const extractTags = dataset => {
   }
 }
 
+export const filterAndNormalizeDatasets = datasets => {
+  const consortiumsToInclude = getEnabledBrand().catalogConsortiumsToInclude
+  const filteredDatasets = _.filter(dataset => consortiumsToInclude === undefined ||
+      _.intersection(consortiumsToInclude, _.map(dataCollection => dataCollection && dataCollection['dct:title'], dataset['TerraDCAT_ap:hasDataCollection'])).length > 0,
+  datasets.result || [])
```

Comment:
Instead of checking `consortiumsToInclude` in every invocation of the predicate function, it could be checked once and the predicate function switched out based on it.

Also, `_.map('key', list)` is a shorthand for `_.map(item => item['key'], list)` and Lodash handles falsy values so there's not a need for `datasets.result || []`.

```suggestion
  const filteredDatasets = _.filter(constortiumsToInclude ?
      dataset => _.intersection(consortiumsToInclude, _.map('dct:title', dataset['TerraDCAT_ap:hasDataCollection'])).length > 0 :
      _.constant(true),
  datasets.result)
```

Reply:
Or... there's no need to run the filter at all if there's no consortia to include.

```suggestion
  const filteredDatasets = constortiumsToInclude ?
    _.filter(dataset => _.intersection(consortiumsToInclude, _.map('dct:title', dataset['TerraDCAT_ap:hasDataCollection'])).length > 0, datasets.result) :
  datasets.result
```